File: ScoreFrame/ScoreFrame/Models/Team.swift
import Foundation

enum Team: String, Codable, CaseIterable, Identifiable {
    case home
    case away

    var id: String { rawValue }
}

struct PKKick: Codable, Identifiable, Equatable {
    var id: UUID = UUID()
    var team: Team
    var order: Int
    var isGoal: Bool
    var timestamp: TimeInterval
}

File: OverScore-Android/app/src/main/java/com/overscore/model/Team.kt
package com.overscore.model

import kotlinx.serialization.Serializable

@Serializable
enum class Team {
    home,
    away;
}

@Serializable
data class PKKick(
    val id: String = java.util.UUID.randomUUID().toString(),
    val team: Team,
    val order: Int,
    val isGoal: Boolean,
    val timestamp: Double
)
